easybuild/framework/easyblock.py

```diff
@@ -332,7 +337,7 @@ def fetch_patches(self, list_of_patches, extension=False, checksums=None):
                         patchspec['copy'] = suff
                     else:
                         patchspec['sourcepath'] = suff
-                if level:
+                if level is not None:
```

Comment:
how did you hit this exactly? is this case covered by the tests you've added now?